### DISCUSSION TITLE
Why do we have two Web Components packages?

### DISCUSSION THREAD

**Discussion by fterra-encora** [carbon-design-system/carbon-for-ibm-dotcom#10697] [Q&A] [answered]
I would like to know why there are two Web Components packages, namely:
* one at [carbon-web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/carbon-web-components), installable as `@carbon/web-components`;
* another one at [web-components](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/main/packages/web-components), installable as `@carbon/ibmdotcom-web-components`.

Mainly, which one should I use to build my web app?

**Comment by kennylam** [comment, marked as answer]
@fterra-encora `@carbon/web-components` are based off `@carbon/react`, and follow its lead in design and functional specs. They are a web components version of the React package, that's all. Both of these packages are mostly pure UI components.

`@carbon/ibmdotcom-web-components`, on the other hand, are components meant to be used for pages on IBM.com only. Its components are more editorial in nature, suitable for product and marketing pages. It also has some global components, such as masthead navigation and the footer. `@carbon/web-components` is one of its main dependencies.

If you are not building for IBM.com, you will want to use `@carbon/web-components`. We just recently released the v2 release candidate (use `v2-next` tag), which uses Carbon v11 and has full parity with the latest version of `@carbon/react`.
